mapping.json

```diff
@@ -1,38 +1,48 @@
+        { "regexRule": "google-analytics\\.com\\/cx\\/api\\.js", "surrogate": "api.js" },
+        { "regexRule": "google-analytics\\.com\\/gtm\\/js", "surrogate": "gtm.js" }
     ],
     "googletagmanager.com": [
-        ["googletagmanager.com/gtm.js", "gtm.js"]
+        { "regexRule": "googletagmanager\\.com\\/gtm\\.js", "surrogate": "gtm.js" }
     ],
     "googletagservices.com": [
-        ["googletagservices.com/gpt.js", "gpt.js"],
-        ["googletagservices.com/tag/js/gpt.js", "gpt.js"]
-    ],        
+        { "regexRule": "googletagservices\\.com\\/gpt\\.js", "surrogate": "gpt.js" },
+        { "regexRule": "googletagservices\\.com\\/tag\\/js\\/gpt\\.js", "surrogate": "gpt.js" }
+    ],
     "googlesyndication.com": [
-        ["googlesyndication.com/adsbygoogle.js", "adsbygoogle.js"],
-        ["googlesyndication.com/pagead/js/adsbygoogle.js", "adsbygoogle.js"]
+        { "regexRule": "googlesyndication\\.com\\/adsbygoogle\\.js", "surrogate": "adsbygoogle.js" },
+        { "regexRule": "googlesyndication\\.com\\/pagead\\/js\\/adsbygoogle\\.js", "surrogate": "adsbygoogle.js" }
     ],
     "doubleclick.net": [
-        ["doubleclick.net/instream/ad_status.js", "ad_status.js"],
-        ["doubleclick.net/tag/js/gpt.js", "gpt.js"]
+        { "regexRule": "doubleclick\\.net\\/instream\\/ad_status\\.js", "surrogate": "ad_status.js" },
+        { "regexRule": "doubleclick\\.net\\/tag\\/js\\/gpt\\.js", "surrogate": "gpt.js" }
     ],
     "scorecardresearch.com": [
-        ["scorecardresearch.com/beacon.js", "beacon.js"]
+        { "regexRule": "scorecardresearch\\.com\\/beacon\\.js", "surrogate": "beacon.js" }
     ],
     "outbrain.com": [
-        ["outbrain.com/outbrain.js", "outbrain.js"]
+        { "regexRule": "outbrain\\.com\\/outbrain\\.js", "surrogate": "outbrain.js" }
     ],
     "amazon-adsystem.com": [
-        ["amazon-adsystem.com/aax2/amzn_ads.js", "amzn_ads.js"]
+        { "regexRule": "amazon-adsystem\\.com\\/aax2\\/amzn_ads\\.js", "surrogate": "amzn_ads.js" }
     ],
     "chartbeat.com": [
-        ["chartbeat.com/chartbeat.js", "chartbeat.js"],
-        ["chartbeat.com/js/chartbeat.js", "chartbeat.js"]
+        { "regexRule": "chartbeat\\.com\\/chartbeat\\.js", "surrogate": "chartbeat.js" },
+        { "regexRule": "chartbeat\\.com\\/js\\/chartbeat\\.js", "surrogate": "chartbeat.js" }
+    ],
+    "facebook.net": [
+        { "regexRule": "facebook\\.net\\/[a-z_A-Z]+\\/sdk\\.js", "surrogate": "fb-sdk.js", "action": "block-ctl-fb" },
+        { "regexRule": "facebook\\.net\\/[a-z_A-Z]+\\/all\\.js", "surrogate": "fb-sdk.js", "action": "block-ctl-fb" }
+    ],
+    "facebook.com": [
+        { "regexRule": "facebook\\.com\\/[a-z_A-Z]+\\/sdk\\.js", "surrogate": "fb-sdk.js", "action": "block-ctl-fb" }
+    ],
+    "youtube.com": [
+        { "regexRule": "youtube(-nocookie)?\\.com\\/iframe_api", "surrogate": "youtube-iframe-api.js", "action": "block-ctl-yt" }
```

Comment:
Can you split this into two entries, one for youtube.com and one for youtube-nocookie.com